package/sdbus-cpp/sdbus-cpp.mk

```diff
@@ -0,0 +1,26 @@
+################################################################################
+#
+# SDBUS_CPP
+#
+################################################################################
+
+SDBUS_CPP_VERSION = 0.8.2
+SDBUS_CPP_SITE = $(call github,Kistler-Group,sdbus-cpp,v$(SDBUS_CPP_VERSION))
+SDBUS_CPP_LICENSE = LGPL-2.1
+SDBUS_CPP_LICENSE_FILES = COPYING
+SDBUS_CPP_DEPENDENCIES = host-pkgconf systemd
```

Comment:
why is this not dependent on the host-sdbuss-cpp?

Reply:
The host tool is a code generator. If we use it in our emb-core build, we'll need it available.